modules/serverless-creating-broker-annotation.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-using-brokers.adoc
+
+[id="serverless-creating-broker-annotation_{context}"]
+= Creating a broker by annotating a trigger
+
+[IMPORTANT]
+====
+If you create a broker by using the `eventing.knative.dev/injection: enabled` annotation, you cannot delete this broker without cluster administrator permissions.
+If you delete the broker without having a cluster administrator remove this annotation first, the broker will be created again after deletion.
+====
+
+.Prerequisites
+* The {ServerlessOperatorName} and Knative Eventing are installed on your {product-title} cluster.
+
+.Procedure
+* Create a `Trigger` YAML object that has the `eventing.knative.dev/injection: enabled` annotation:
++
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  annotations:
+    eventing.knative.dev/injection: enabled
+  name: example-trigger <1>
+spec:
+  broker: default <2>
+  subscriber: <3>
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: my-service
+----
++
+<1> The name of the trigger.
+<2> The name of the broker where events will be filtered from. If the broker is not specified, the trigger will revert to using the `default` broker.
```

Comment:
"If the broker is not specified, the trigger will revert to using the `default` broker."

this is no longer true for v1 Trigger , broker needs to be set explicitly now.